admin/page-assets-metabox.php

```diff
@@ -0,0 +1,132 @@
+<?php
+/**
+ * Handles creating the page assets metaboxes
+ **/
+if ( ! class_exists( 'UCF_Page_Assets_Metabox' ) ) {
+    class UCF_Page_Assets_Metabox {
+
+		/**
+		 * Enqeues admin assets
+		 * @author Jim Barnes
+		 * @since 1.0.0
+		 * @param $hook string | The current admin hook
+		 **/
+		public static function enqueue_assets( $hook ) {
+			if ( 'post.php' === $hook ) {
+				wp_enqueue_script(
+					'ucf-page-assets-js',
+					UCF_PAGE_ASSETS__JS_URL . '/ucf-page-assets.min.js',
+					array( 'jquery' ),
+					null,
+					true
+				);
+			}
+		}
+
+		/**
+		 * Adds the Custom Page Assets metabox
+		 * @author Jim Barnes
+		 * @since 1.0.0
+		 **/
+		public static function add_meta_box() {
+			add_meta_box(
+				'ucf-page-assets',
+				__( 'Custom Page Assets' ),
```

Comment:
It might make sense to remove references to Pages where we're describing these custom fields, in case they're ever used on other custom post types